yarf/rf_libraries/resources/kvm.resource

```diff
@@ -340,3 +340,35 @@ Ensure ${destination} Does Not Match
     IF    ${match}
         Fail                    Found ${destination} where it wasn't intended or expected
     END
+
+Drag ${target} And Drop On ${destination}
```

Comment:
Unfortunately this keyword name is not compatible with the other below, or vice-versa. I think we should keep just the other one, asking the user to move the pointer first by themselves